### PR TITLE
fix: fix built ITs Docker images job

### PR DESCRIPTION
**Pull request by kuisathaverat**
## What does this PR do?

Finish the implementation of the pipeline to build ITs Docker images.

## Why is it important?

Before this PR, the job was not finished and contains some errors.

## Checklist

- ~~[ ] My code follows the style guidelines of this project~~
- ~~[ ] I have commented my code, particularly in hard-to-understand areas~~
- ~~[ ] I have made corresponding changes to the documentation~~
- ~~[ ] I have made corresponding change to the default configuration files~~
- ~~[ ] I have added tests that prove my fix is effective or that my feature works~~
- ~~[ ] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.~~

### CODE REVIEW

File: .ci/build-docker-images.groovy [hunk 1]
```diff
@@ -36,24 +34,25 @@ pipeline {
...
     }
-    stage('Install dependencies') {
-      when {
-        expression { return params.RELEASE_TEST_IMAGES }
-      }
-      steps {
-        sh(label: 'Install virtualenv', script: 'pip install --user virtualenv')
-      }
-    }
+    // stage('Install dependencies') {
```
Comment (kuisathaverat) [inline .ci/build-docker-images.groovy:48]: if the build ends correctly, it'd mean that is not need

Comment (jsoriano) [reply .ci/build-docker-images.groovy:48]: You are right, this shouldn't be needed now with python 3.

File: .ci/build-docker-images.groovy [hunk 2]
```diff
@@ -36,24 +34,25 @@ pipeline {
...
+    //   when {
+    //     expression { return params.RELEASE_TEST_IMAGES }
+    //   }
+    //   steps {
+    //     sh(label: 'Install virtualenv', script: 'pip install --user virtualenv')
+    //   }
+    // }
```
Comment (kuisathaverat) [inline .ci/build-docker-images.groovy:55]:
```suggestion
```

**Comment by kuisathaverat** [comment]
green https://beats-ci.elastic.co/blue/organizations/jenkins/Beats%2Fbuild-it-docker-images/detail/build-it-docker-images/23/pipeline/

cc @jsoriano

**Pull request merged**